### ISSUE TITLE
hello!

### ISSUE THREAD

**Issue by youwantsy** [lzhangbj/DualVar#1]
thank you for your great research! 
can you explain me your experiment setting in Pre-training and Fine-tuning?
i can't reach the accuracy presented in this paper. 
( UCF-101[pre-train] to UCF-101[fine-tune] in MoCo, SimCLR, (in this paper, Table1.))
My accuracy is 55% ~ 57%. i want to know your experiment setting. After that, i would like to compare if there is anything else.

thank you.
